e2e-test/README.md

```diff
@@ -0,0 +1,57 @@
+# End-to-end Tests
+
+This folder contains end-to-end tests which test the whole functionality of the application.
+
+## Prerequisites
+
+Before running the e2e-tests you should make sure that the project is running and that it is provisioned. If you have questions on how to start the project please visit the [Developer-Setup](https://github.com/openkfw/TruBudget/blob/master/doc/tutorials/contribute/Developer-Setup.md#developer-setup/)
+
+## Setup
```

Comment:
I would suggest to move the setup entirely to the developer setup guide because e2e-tests are only executed if someone is developing/contributing. Also the env vars are not correct 
-> EXPORT_SERVICE_BASE_URL=http://localhost:8888 is missing on some lines